docs/jobs/openings/web3.md

```diff
@@ -0,0 +1,34 @@
+## Web3 Developer Opening at Aragon
+
+Aragon is an open source, decentralized project fighting for the freedom to organize. We believe that decentralized organizations can solve the world’s worst problems.
+
+We’re looking for a Web3 Developer to help us consolidate smart contracts with a frontend, we call this position The Glue. Our Web 3 Developer must not only be a Javascript rockstar but also a decentralization evangelist, someone who enjoys researching and testing new Web3 technologies.
+
+### Responsibilities
+
+- Take ownership of the development and maintenance of [aragon.js](https://github.com/aragon/aragon.js), the JavaScript library between smart contracts and the native web experience.
+- Research and experiment with new Web3 technologies, including but not limited to [Web3.js](https://github.com/ethereum/web3.js/) and [Parity systems](https://github.com/paritytech).
+- Assist on smart contract and frontend development when needed.
+- Help create developer tools and technical documentation.
```

Comment:
I'd remove this and the point above to instead add "Assist others with Web3-related development when needed" to broaden the scope to be encompassing all things related to Web3.